### DISCUSSION TITLE
skywalking webapp ui Search logs based on content keywords

### DISCUSSION THREAD

**Discussion by dresstokill** [apache/skywalking#12467] [Q&A] [answered]
![日志详情](https://github.com/user-attachments/assets/475b2905-0170-4a80-9929-a7c61300bdcb)
![内容关键词搜索](https://github.com/user-attachments/assets/70e86782-97ee-4899-863c-172d9b8ef2ad)

I want to directly use the log search page, as shown in the log details in the picture.
The content keywords use the entire word 'error1721631824622', and the search results are correct. However, if the content keyword "24622" is used, the log cannot be found. Does skywalking not support querying for non whole words?

**Comment by wu-sheng** [comment, marked as answer]
Learn what is Elasticsearch text analyzer. This is a simple fuzzy query
